File: 1392-find-the-difference-of-two-arrays/find-the-difference-of-two-arrays.cpp
class Solution {
public:
    vector<vector<int>> findDifference(vector<int>& nums1, vector<int>& nums2) {
        unordered_map<int,int>map1;
        unordered_map<int,int>map2;
        for(int i:nums1){map1[i]++;}
        for(int i:nums2){map2[i]++;}
        vector<int>a,b;
        for(auto &i:map1){
            if(map2[i.first]==0){
                a.push_back(i.first);
            }
        }
        for(auto &i:map2){
            if(map1[i.first]==0){
                b.push_back(i.first);
            }
        }
        return {a,b};
    }
};

File: 1392-find-the-difference-of-two-arrays/find-the-difference-of-two-arrays.py
class Solution:
    def findDifference(self, nums1: list[int], nums2: list[int]) -> list[list[int]]:
        s1=set(nums1)
        s2=set(nums2)
        right=[]
        left=[]
        for i in s2:
            if i not in s1:
                right.append(i)

        for i in s1:
            if i not in s2:
                left.append(i)

        return [left,right]